ui/raidboss/data/06-ew/alliance/thaleia.ts

```diff
@@ -0,0 +1,629 @@
+    },
+    {
+      id: 'Thaleia Oschon Trek Shot',
+      type: 'StartsUsing',
+      netRegex: { id: '898E', source: 'Oschon', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to side of the arrow (Boss doesn\'t move)', // FIXME
+          ja: '矢印の横へ (ボスは動かない)',
+        },
+      },
+    },
+    {
+      id: 'Thaleia Oschon Reproduce',
+      type: 'StartsUsing',
+      netRegex: { id: '8989', source: 'Oschon', capture: false },
+      alertText: (data, _matches, output) => {
+        if (!data.soaringMinuet)
+          return output.one!();
+        return output.two!();
+      },
+      outputStrings: {
+        one: {
+          en: 'Go to side of the arrow', // FIXME
+          ja: '矢印の横へ',
+        },
+        two: {
+          en: 'Corner between two arrows', // FIXME
+          ja: '2つの矢印の隅',
+        },
+      },
+    },
+    {
+      id: 'Thaleia Oschon Flinted Foehn',
+      type: 'StartsUsing',
+      netRegex: { id: ['89A3', '89A4'], source: 'Oschon', capture: false },
+      suppressSeconds: 5,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'AOE',
+          ja: '連続AOE',
+        },
+      },
+    },
+    {
+      id: 'Thaleia Oschon Soaring Minuet',
+      type: 'StartsUsing',
+      netRegex: { id: ['8D0E', '8994'], source: 'Oschon', capture: false },
+      response: Responses.getBehind(),
+      run: (data) => data.soaringMinuet = true,
+    },
+    {
+      id: 'Thaleia Oschon the Arrow',
+      type: 'StartsUsing',
+      netRegex: { id: ['899B', '899C', '899D', '899E'], source: 'Oschon' },
+      suppressSeconds: 10,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Thaleia Oschon Climbing Shot',
+      type: 'StartsUsing',
+      netRegex: { id: ['8990', '8991', '8992', '8993'], source: 'Oschon', capture: false },
+      suppressSeconds: 10,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Knockback to safe corner', // FIXME
+          ja: 'AOEがないどころへ + ノックバック',
+        },
+      },
+    },
+    {
+      id: 'Thaleia Oschon Lofty Peaks',
+      type: 'StartsUsing',
+      netRegex: { id: '89A7', source: 'Oschon', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Oschon second phase',
+          ja: 'すぐ大きくなる',
+        },
+      },
+    },
+    /*
+    {
+      id: 'Thaleia Oschon Piton Pull',
+      type: 'StartsUsing',
+      netRegex: { id: ['89A9', '89AA', '89AB'], source: 'Oschon', capture: false },
+      suppressSeconds: 5,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to safe zone between two hooks', // FIXME
+          ja: '2つの鉤の隅へ',
+        },
+      },
+    },
+    */
```

Comment:
Can you delete this or just leave a one line TODO at the top of the file rather than a full commented out block?